src/modules/filelist/FileListHeaderDesktop.jsx

```diff
@@ -29,20 +30,22 @@ const FileListHeaderDesktop = ({
 }) => {
   const { t } = useI18n()
   const actualSort = sort || DEFAULT_SORT
+  const isTwake = isTwakeTheme()
 
   return (
     <TableHead className={styles['fil-content-head']}>
       <TableRow className={styles['fil-content-row-head']}>
         <TableHeader
           className={cx(
             styles['fil-content-header'],
-            styles['fil-content-file-select']
+            styles['fil-content-file-select'],
+            { [styles['u-capitalize']]: isTwake }
```

Comment:
it will be a bit confusing to use `u-something` as class name due to using this keyword for utility classes. We can use here something like `fil-content-header--capitalize` maybe